.github/workflows/ci.yml

```diff
@@ -173,4 +172,4 @@ jobs:
           (echo "$PGP_PASSPHRASE"; echo; echo) | gpg --command-fd 0 --pinentry-mode loopback --change-passphrase $(gpg --list-secret-keys --with-colons 2> /dev/null | grep '^sec:' | cut --delimiter ':' --fields 5 | tail -n 1)
 
       - name: Publish
-        run: sbt '++ ${{ matrix.scala }}' tlCiRelease
+        run: sbt tlCiRelease
```

Comment:
> do you know why this is messing around with the build matrix?

Yes, this is a fix :)

- https://github.com/typelevel/sbt-typelevel/issues/502